app/scripts/context.js

```diff
@@ -0,0 +1,7 @@
+export const METAMASK_CONTENTSCRIPT = 'metamask-contentscript';
```

Comment:
Is it worth 1) using TypeScript for this file and 2) using an enum for these values?

Reply:
fixed 01dd34ba7